heartbeat/monitors/wrappers/monitorstate/tracker_test.go

```diff
@@ -131,3 +131,37 @@ func TestDeferredStateLoader(t *testing.T) {
 	resState, _ = dsl(stdfields.StdMonitorFields{})
 	require.Equal(t, stateA, resState)
 }
+
+func TestStateLoaderRetry(t *testing.T) {
+	tests := []struct {
+		name          string
+		retryable     bool
+		expectedCalls int
+	}{
+		{
+			"should retry 3 times when fails with retryable error",
```

Comment:
this test lasts 5s given that a time.Sleep is executed for every retry. Let me know if you want me to make this parameterizable or if you believe it's something negligible.

Reply:
Maybe we should make it configurable.